docs/known_generator_issues.md

```diff
@@ -7,3 +7,16 @@ error CS8138: Cannot reference 'System.Runtime.CompilerServices.TupleElementName
 ```
 
 Workaround: update the declaration using tuple syntax to define tuple names. Or comment/remove this attribute.
+
+## Circular dependency between reference package and targeting pack
+
+```text
+error NU1108: Cycle detected.
+error NU1108:   <Reference project> -> NETStandard.Library x.y.z -> <Reference project>
+```
+
+Workaround: add the following property to the project.
+
+```xml
+<DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
```

Comment:
This setting should only be enabled for netstandard1.x TFMs. It's fine to use StartsWith here as netstandard1.x will never use platforms.

```suggestion
<DisableImplicitFrameworkReferences Condition="$(TargetFramework.StartsWith('netstandard1'))">true</DisableImplicitFrameworkReferences>
```